### PR TITLE
#39490 - Forgot password

### PR DESCRIPTION
**Pull request by Aekal**
Internal: 39490

### CODE REVIEW

File: Magento_Captcha/templates/default.phtml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            "captcha": {
+                "url": "<?= $block->escapeUrl($block->getRefreshUrl()) ?>",
+                "imageLoader": "<?= $block->escapeUrl($block->getViewFileUrl('images/loader-2.gif')) ?>",
+                "type": "<?= $block->escapeHtmlAttr($block->getFormId()) ?>"
+            }
+        }'
+    >
+        <img
+            alt="<?= /* @escapeNotVerified */ __('Please type the letters below') ?>"
+            class="captcha-img image lazyload forgot-password__image"
```
Comment (anqaka) [inline Magento_Captcha/templates/default.phtml]: I'd split classes here...

File: Magento_Captcha/templates/default.phtml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            height="<?= /* @noEscape */ (float) $block->getImgHeight() ?>"
+            data-src="<?= $block->escapeUrl($captcha->getImgSrc()) ?>"
+            src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw"
+        />
+        <button
+            type="button"
+            class="button button--secondary reload captcha-reload"
```
Comment (anqaka) [inline Magento_Captcha/templates/default.phtml]: it's good place to split classes too :)

File: Magento_Customer/templates/form/forgotpassword.phtml [hunk 3]
```diff
@@ -1,33 +1,33 @@
...
-      action="<?= /* @escapeNotVerified */ $block->getUrl('*/*/forgotpasswordpost') ?>"
-      method="post"
-      id="form-validate"
-      data-mage-init='{"validation":{}}'
->
-    <p class="paragraph">
+<div class="forgot-password">
+    <p>
         <?= /* @escapeNotVerified */ __('Please enter your email address below to receive a password reset link.'); ?>
     </p>
```
Comment (anqaka) [inline Magento_Customer/templates/form/forgotpassword.phtml]: I have some proposition to make this code even more accessible:  you can add to this `<p>` an `id` attribute and use it as a value for an attribute `aria-labelledby` in `<form>`, you'll have additional tips for screen reader. 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

File: Magento_Captcha/templates/default.phtml [hunk 4]
```diff
@@ -0,0 +1,50 @@
+<?php
+/** @var \Magento\Captcha\Block\Captcha\DefaultCaptcha $block */
+/** @var \Magento\Captcha\Model\DefaultModel $captcha */
+$captcha = $block->getCaptchaModel();
+?>
+<div role="<?= $block->escapeHtmlAttr($block->getFormId()) ?>">
```
Comment (anqaka) [inline Magento_Captcha/templates/default.phtml]: I'm curious about that `role` attribute. This is the custom one. I know that HTML5 validate `data-` attributes but `role ` attribute has some concrete values and when I did an accessibility audit of this page, it informed me that is not valid:
screen shot 2018-03-27 at 17 51 29
could you please check it? If we can change it or is Magento issue? we shouldn't have invalid HTML ;)

File: Magento_Captcha/templates/default.phtml (hunk 2 above)
Comment (Aekal) [reply Magento_Captcha/templates/default.phtml]: Done

File: Magento_Customer/templates/form/forgotpassword.phtml (hunk 3 above)
Comment (Aekal) [reply Magento_Customer/templates/form/forgotpassword.phtml]: Great, thanks!

File: Magento_Captcha/templates/default.phtml (hunk 4 above)
Comment (Aekal) [reply Magento_Captcha/templates/default.phtml]: It's just a default magento error ;) I removed the entire div

**Comment by Aekal** [comment]
@anqaka The captcha img should be smaller, I think it's fine, maybe you didn't apply styles from components

File: Magento_Captcha/templates/default.phtml (hunk 4 above)
Comment (anqaka) [reply Magento_Captcha/templates/default.phtml]: great 👍

**Pull request merged**